1-getting-started/lessons/3-sensors-and-actuators/translations/assignment.hi.md

```diff
@@ -0,0 +1,17 @@
+# रिसर्च सेंसर और एक्चुएटर्स
+
+## निर्देश
+
+इस पाठ में सेंसर और एक्चुएटर्स शामिल थे। अनुसंधान और एक सेंसर और एक एक्चुएटर का वर्णन करें जिसका उपयोग IoT देव किट के साथ किया जा सकता है, जिसमें शामिल हैं:
```

Comment:
I think this (anusandhan karen aur sensors tatha actuactors ka .....) will be much better or something else as it seems little confusing